doc/ref_model/chapters/chapter01.md

```diff
@@ -238,7 +243,98 @@ The ratings are as follows:
 
 <a name="1.9"></a>
-## 1.9 Roadmap
+## 1.9 Networking and Fabric Strategy
+
+> _**Editor's Note:** The purpose in the subsections comprising RM 1.9 is to provide initial high-level content related to networking across CNTT, so as to provide general direction and help coordinate current independent networking development activities, as well as to satisfy the networking MVP deliverables for the Baldy release. The following content addresses the overall CNTT networking solution, and **is not solely focused on the RM**. Hence, post-Baldy, it is anticipated this content will be expanded and be relocated where appropriate._
+
+The basic approach to producing the CNTT Baldy deliverables is to:
+A) Summarize the overall CNTT Networking and Fabric Strategy
+B) Document the initial list of Objectives and Requirements
+C) Document enough initial logistical details for contributors to create coherent content
+
+Readers be aware, in parallel with the CNTT Networking Strategy, the RI team is implementing networking for use today in labs and by RC. As RI has to deliver a working network at the same time the initial networking strategy and specifications (i.e. the work related to RM 1.9) are to be delivered, it is not expected nor mandated RI networking be compliant in the immediate future. However, the RI will need to be compatible, meaning, it delivers the network connectivity required by the RA and by RC, even if it does not implement the APIs, topology, encapsulation, etc., that will ultimately be specified herein.
+
+The following subsections, Executive Summary, Strategy Objectives, Networking Requirements and Initial Approach, respectively, are intended to provide an overview of CNTT's vision for networking and intent, objectives, requirements and their supporting rationale, as well as initial development approach.
+
+The networking within an NFVI, fabric or otherwise, is an area where there is significant variability across implementations. Leaf-Spine topology is well established, however, after topology there are countless decisions an Operator needs to make. Differences arise from many aspects, for example, is the solution layer-2 or layer-3; is the routing static or dynamic; what mechanism is used for encapsulation; what mechanism is used for isolation; does it support SR-IOV; does it support DPDK; does it employee SmartNICs; does it employ distributed control or a centralized control driving a programmable fabric; and the list continues. The multitude of permutations enable Operators to design (or procure) a fabric/networking solution that's optimized for their needs, whether their needs are minimal, very extensive or somewhere in between.
+
+For CNTT, a strategy is needed that affords Operators the performance, flexibility, availability, maintainability and scalability their business requires, yet doesn't require OPNFV to design, manage and test prohibitive numbers of networking solutions. // relates to Petar's #2  
```

Comment:
Request removal of the comment referring to a CNTT contributor

Reply:
Done
Note: Multiple reference deleted